src/scenario.h

```diff
@@ -302,7 +302,7 @@ typedef struct {
 	char custom_strings[0x8000];
 	uint32 game_ticks_1;
 	rct_ride rides[255];
-	uint16 word_01388698;
+	uint16 word_01388698; // TODO: rename to match RCT2_ADDRESS_SAVED_AGE ???
```

Comment:
Saved age isn't a particular good name, it seems to be ticks the scenario has ran for, it gets reset in reset_sprite_list...